modules/cli/pages/mctestauth.adoc

```diff
@@ -0,0 +1,93 @@
+= mctestauth
+:description: pass:q[The mctestauth tool allows you to troubleshoot authentication issues in data services.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+== Description
+
+The `mctestauth` tool allows you to troubleshoot authentication issues in data services. Data services use the Simple Authentication and Security Layer (SASL) framework for authentication. In addition to its own SASL authentication mechanisms, the data services allow the usage of any authentication mechanisms supported by SASL. For example, SCRAM-SHA512 or PLAIN. 
```

Comment:
I don't understand the last part: 

    In addition to its own SASL authentication mechanisms, the data services allow the usage
    of any authentication mechanisms supported by SASL. For example, SCRAM-SHA512 or PLAIN. 

What is our own SASL authentication mechanisms? We use SASL, and as part of SASL the client may ask the server for a list of authentication mechanisms it support.